tests/conftest.py

```diff
@@ -121,20 +124,25 @@ def redis():
 
 @pytest.fixture
 def cache(redis):
-    from dataops.dependencies import Cache
-
     yield Cache(redis)
 
 
-@pytest_asyncio.fixture
-async def test_client(db_session, redis):
+@pytest.fixture
+def app(event_loop, db_session, cache, redis) -> FastAPI:
     async def replace_cache():
-        return Cache(redis)
+        return cache
 
-    from dataops.app import create_app
+    async def replace_redis():
+        return redis
 
     app = create_app()
     app.dependency_overrides[get_db_session] = lambda: db_session
     app.dependency_overrides[CacheInstance.get_cache] = replace_cache
+    app.dependency_overrides[get_redis_connection] = replace_redis
+    yield app
+
+
+@pytest_asyncio.fixture
```

Comment:
There's not need to use `pytest_asyncio` anymore.
```suggestion
@pytest.fixture
```